doc/classes/AnimatedSprite2D.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Animations are created using a [SpriteFrames] resource, which can be configured in the editor via the SpriteFrames panel.
+		[b]Note:[/b] You can associate a set of normal or specular maps by creating additional [SpriteFrames] resources with a [code]_normal[/code] or [code]_specular[/code] suffix. For example, having 3 [SpriteFrames] resources [code]run[/code], [code]run_normal[/code], and [code]run_specular[/code] will make it so the [code]run[/code] animation uses normal and specular maps.
```

Comment:
This is incorrect (or at least a confusing wording). In the screenshot in #38509, `_normal` is an animation in the same resource, not another `SpriteFrames` resource.